neurons/remote_config.py

```diff
@@ -50,7 +51,9 @@ def load_remote_config(self):
                     response = requests.get(self.config_url, timeout=10)
                     response.raise_for_status()
                     self.config_cache = response.json()
-
+                                        
+                    self.config_cache.update(self.fetch_inmemory_hotkeys(current_netuid=15))
```

Comment:
can we use netui from cofig ?

Reply:
hmm, i think need to add netuid to the load_and_get_config_values. anyway let me try and update.